sphinx/builders/latex/theming.py

```diff
@@ -111,10 +111,10 @@ def load_builtin_themes(self, config: Config) -> None:
 
     def get(self, name: str) -> Theme:
         """Get a theme for given *name*."""
-        if name in self.themes:
-            theme = self.themes[name]
-        else:
-            theme = self.find_user_theme(name) or Theme(name)
+        theme = self.themes.get(
+            name,
+            self.find_user_theme(name) or Theme(name)
+        )
```

Comment:
Most are fine, I don't like this one though, due to the `or` in the `default` argument.